src/worker/read-wordlist.ts

```diff
@@ -38,6 +38,40 @@ export async function readExcel(
   return wordlist;
 }
 
+/**
+ * Function to read a TSV file with logic from keymanapp
+ * @param TSVFile UTF-8 String
+ */
+export function readTSV(TSVFile: string): WordList {
+  const wordlist: WordList = [];
+  // get file contents
+  const rows = TSVFile.split(/\u000d?\u000a/);
+  for (let i = 0; i < rows.length; i++) {
+    // Remove BOM from beginning of the string
+    rows[i] = rows[i].replace(/^\uFEFF/, "");
+
+    if (rows[i].startsWith("#") || rows[i] === "") {
+      continue; // skip comments and empty lines
+    }
+
+    // eslint-disable-next-line prefer-const
+    let [word, countText, _comment] = rows[i].split("\t");
```

Comment:
Why is there this eslint error?

Honestly, I disagree with the eslint rule of prefer `const` over `let` because `const` is misleading, haha! But regarless, why not just change it to `const`?

Also, (I think I wrote the original code here, haha!) but to sush the warning:

```suggestion
    const [word, countText, /* comment */] = rows[i].split("\t");
```

Reply:
word and countText are reassigned on the following lines so changing to const will cause an error. Pushing a small refactor to avoid the eslint complaints